app/messages/latest_message.rb

```diff
@@ -0,0 +1,23 @@
+class LatestMessage < SlackMessage
+  values do
+    attribute :last_deployment, Deployment
+  end
+
+  def to_message
```

Comment:
GitHub now has UI for viewing deployment activity. For example, here's the latest deployments for this repo: https://github.com/remind101/slashdeploy/deployments

I wonder if we should add a footer to this message that links there?

Reply:
Absolutely! makes sense to add a link to github UI, I've added it on the latest commit, tell me if the message looks good or if I should change it 👍